articles/quickstarts/install-command-line.md

```diff
@@ -18,7 +18,12 @@ Q# programs can be executed on their own, without a driver in a host language li
 
 ## Installation
 
-While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+While you can build Q# command line applications in any IDE, we recommend using Visual Studio Codespaces, Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
```

Comment:
Looks good overall. Do we want to make Codespaces the first option though, as that would give people an impression that is it promoted more as a default? Also we should briefly describe the difference between these environments (like this page does https://docs.microsoft.com/en-us/quantum/quickstarts/) especially given Codespaces requires an Azure account, would be good to call out the other two options do not require it.

Reply:
Great point! I've added a distinction as VS Code/Visual Studio being for local development and VS Codespaces as a cloud/browser option. I also linked on how to use VS Code to a Codespaces environment.